corehq/couchapps/__init__.py

```diff
@@ -6,4 +6,6 @@
     'users_extra': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
     'noneulized_users': (settings.USERS_GROUPS_DB, settings.NEW_USERS_GROUPS_DB),
     'all_docs': (None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB),
+    # TODO should we make an '_all' option or something?
+    'by_domain_doc_type': (None, settings.NEW_USERS_GROUPS_DB, settings.NEW_FIXTURES_DB),
```

Comment:
I'm thinking about how to future-proof against subsequent migrations - seems like `all_docs` and `by_domain_doc_type` should exist in ~every db.